core/src/main/java/org/apache/iceberg/io/BaseTaskWriter.java

```diff
@@ -115,6 +116,13 @@ protected BaseEqualityDeltaWriter(StructLike partition, Schema schema, Schema de
      */
     protected abstract StructLike asStructLike(T data);
 
+    /**
+     * Convert a {@link StructLike} into {@linkplan T}
```

Comment:
What is `linkplan`?

Reply:
That's meant to be `linkplain` but I can update it. This is still a WIP (I removed it from draft as I thought that would make tests run).